src/app/core/store/documents/documents.state.ts

```diff
@@ -46,7 +46,7 @@ export const selectDocumentsByQuery = createSelector(selectAllDocuments, selectQ
 );
 
 export const selectDocumentsByCustomQuery = (query: QueryModel) => createSelector(selectAllDocuments,
-  (documents): DocumentModel[] => filterDocuments(sortDocumentsByCreationDate(documents), query)
+  (documents): DocumentModel[] => filterDocuments(sortDocumentsByCreationDate(documents, true), query)
```

Comment:
It would make more sense to have this parametrized so you can pass this order as optional parameter to this selector.